bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/BlueZBluetoothDevice.java

```diff
@@ -52,6 +53,8 @@
 
     private final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("bluetooth");
 
+    private final InactiveCleanupJob inactiveCleanupJob = new InactiveCleanupJob();
```

Comment:
The instance should be set in the `initialize()` method.

Reply:
done